### PR TITLE
delete unused code

### PR DESCRIPTION
**Pull request by jinwon0325kim**
In the automation test, the method of command to the things changed, so unused code was deleted.
related https://github.com/Samsung/ioter/pull/51

### CODE REVIEW

File: src/common/device_command.py [hunk 1]
```diff
@@ -85,7 +84,7 @@
 LIGHTSENSOR_MAX_VAL = 3576000
 MEASURED_VALUE_MIN = 1
 MEASURED_VALUE_MAX = 65534
-LIGHTSENSOR_UNIT = ' lux'
+LIGHTSENSOR_UNIT = ' lx'
```
Comment (spdkimo) [inline src/common/device_command.py:87]: SmartThings using lux as brightness unit
Can you explain why do you try to change this?

File: src/things/contact.py [hunk 2]
```diff
@@ -145,7 +127,7 @@ def setContactCmd(self, value):
 
     ## Get contact sensor state ##
     def getContactState(self):
-        return "Close" if self.pushButtonStatus.isChecked() else "Open"
+        return self.toggle_text.get(self.pushButtonStatus.isChecked())
```
Comment (spdkimo) [inline src/things/contact.py:130]: Eliminating literal is a good idea

File: src/common/device_command.py (hunk 1 above)
Comment (jinwon0325kim) [reply src/common/device_command.py:87]: It is shown in the [Connected Home over IP Application Clusters] document as below.
The MeasuredValue attribute represents the illuminance in Lux (symbol lx) as follows:

Comment (jinwon0325kim) [reply src/common/device_command.py]: I restored to lux as before

**Review by junyong-sim — APPROVED** [review]
LGTM

**Review by spdkimo — APPROVED** [review]
LGTM

**Pull request merged**